[2025-02-02T20:48:16.547+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T20:48:16.569+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flights_pipeline.transform_data manual__2025-02-02T20:48:09.725355+00:00 [queued]>
[2025-02-02T20:48:16.578+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flights_pipeline.transform_data manual__2025-02-02T20:48:09.725355+00:00 [queued]>
[2025-02-02T20:48:16.580+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2025-02-02T20:48:16.594+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2025-02-02 20:48:09.725355+00:00
[2025-02-02T20:48:16.599+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4113) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T20:48:16.601+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flights_pipeline', 'transform_data', 'manual__2025-02-02T20:48:09.725355+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/zdag.py', '--cfg-path', '/tmp/tmpg9lbz24i']
[2025-02-02T20:48:16.602+0000] {standard_task_runner.py:72} INFO - Started process 4145 to run task
[2025-02-02T20:48:16.603+0000] {standard_task_runner.py:105} INFO - Job 81: Subtask transform_data
[2025-02-02T20:48:16.646+0000] {task_command.py:467} INFO - Running <TaskInstance: flights_pipeline.transform_data manual__2025-02-02T20:48:09.725355+00:00 [running]> on host d628bd12729b
[2025-02-02T20:48:16.714+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flights_pipeline' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T20:48:09.725355+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T20:48:09.725355+00:00'
[2025-02-02T20:48:16.716+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T20:48:16.793+0000] {logging_mixin.py:190} INFO - Données transformées avec succès.
[2025-02-02T20:48:16.807+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T20:48:16.815+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T20:48:16.817+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flights_pipeline, task_id=transform_data, run_id=manual__2025-02-02T20:48:09.725355+00:00, execution_date=20250202T204809, start_date=20250202T204816, end_date=20250202T204816
[2025-02-02T20:48:16.858+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-02-02T20:48:16.875+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T20:48:16.880+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
